cts/scheduler/summary/on_fail_demote4.summary

```diff
@@ -11,10 +11,12 @@ Current cluster status:
     * Clone Set: rsc1-clone [rsc1] (promotable):
       * rsc1	(ocf:pacemaker:Stateful):	 Promoted rhel7-4 (UNCLEAN)
       * rsc1	(ocf:pacemaker:Stateful):	 Unpromoted remote-rhel7-2 (UNCLEAN)
+      * rsc1	(ocf:pacemaker:Stateful):	 FAILED lxc2
```

Comment:
It's odd that lxc2 is not listed in the node section at all, I'd expect it to be listed as unclean (and lxc-ms to be listed as failed/unclean on it)